From e6b01eead881d58c034eacda29920c695977d5dd Mon Sep 17 00:00:00 2001
From: Ruslan Pisarev <ruslan@rpisarev.org.ua>
Date: Sun, 11 Apr 2010 15:05:51 +0300
Subject: [PATCH 7/8] Staging: rtl8192u: fix space coding style issue in ieee80211_crypt.h

This is a patch to the ieee80211_crypt.h file that fixed up a space
Errors found by the checkpatch.pl tools

Signed-off-by: Ruslan Pisarev <ruslan@rpisarev.org.ua>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/rtl8192u/ieee80211_crypt.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

--- a/drivers/staging/rtl8192u/ieee80211_crypt.h
+++ b/drivers/staging/rtl8192u/ieee80211_crypt.h
@@ -77,7 +77,7 @@ struct ieee80211_crypt_data {
 
 int ieee80211_register_crypto_ops(struct ieee80211_crypto_ops *ops);
 int ieee80211_unregister_crypto_ops(struct ieee80211_crypto_ops *ops);
-struct ieee80211_crypto_ops * ieee80211_get_crypto_ops(const char *name);
+struct ieee80211_crypto_ops *ieee80211_get_crypto_ops(const char *name);
 void ieee80211_crypt_deinit_entries(struct ieee80211_device *, int);
 void ieee80211_crypt_deinit_handler(unsigned long);
 void ieee80211_crypt_delayed_deinit(struct ieee80211_device *ieee,
